### PR TITLE
Update double aggregation

### PR DESCRIPTION
**Pull request by blootsvoets**
Changes:
- Use latest radar-schemas
- full Docker build
- Added source monitor
- integration test fixes

### CODE REVIEW

File: src/integrationTest/java/org/radarcns/integration/E4AggregatedAccelerationMonitor.java [hunk 1]
```diff
@@ -75,11 +74,11 @@ protected void evaluateRecords(ConsumerRecords<GenericRecord, GenericRecord> rec
...
             }
             GenericRecord value = record.value();
-            GenericData.Array count = (GenericData.Array) value.get("count");
+            GenericData.Array fields = (GenericData.Array) value.get("fields");
             logger.info("Received [{}, {}, {}] E4 messages",
-                    count.get(0), count.get(1), count.get(2));
+                    ((GenericRecord)fields.get(0)).get("count"), ((GenericRecord)fields.get(1)).get("count"), ((GenericRecord)fields.get(2)).get("count"));
 
-            if ((Double)count.get(0) > 200) {
+            if ((Integer)((GenericRecord)fields.get(0)).get("count") > 100) {
```
Comment (nivemaham) [inline src/integrationTest/java/org/radarcns/integration/E4AggregatedAccelerationMonitor.java]: What exactly is the number `100`?  Why do we check whether we consumed more than 100 records?

Comment (blootsvoets) [reply src/integrationTest/java/org/radarcns/integration/E4AggregatedAccelerationMonitor.java]: It's a bit arbitrary... Generally, that we get more than 100 records over a 10 second interval.

**Comment by blootsvoets** [comment]
The reason the `AggregateKey` is sent is simply that it was already available in RADAR-Schemas. If needed, I could add more information to this aggregation in a separate schema, but I'm not sure what we'd need.

**Comment by blootsvoets** [comment]
@nivemaham I've added a separate SourceStatistics class to schemas in RADAR-CNS/RADAR-Schemas#91. Could you take a look please?

**Comment by nivemaham** [comment]
@blootsvoets As i gather that there is a `source-statistics-output` topic per `source-type`, i was wondering whether it would be a good idea to incorporate this into the specification file and make it part of the `SourceType` definition. This would help us avoid additional computation to decide on which collection mongodb should look for when a request is sent and maybe makes it flexible. Would that work?

**Comment by blootsvoets** [comment]
@nivemaham can we make it a convention instead? Like `source_statistics_[producer]_[model]`?

**Comment by nivemaham** [comment]
@blootsvoets All right. Also i will make the collection/topic name to lowercase.

**Pull request merged**